zips/zip-0032.rst

```diff
@@ -397,18 +405,25 @@ $\mathsf{MKGh}^\mathsf{Context}(\mathsf{IKM}) \rightarrow (\mathsf{sk}_m, \maths
 Hardened-only child key derivation
 ----------------------------------
 
-$\mathsf{CKDh}^\mathsf{Context}((\mathsf{sk}_{par}, \mathsf{c}_{par}), i)$ $\rightarrow (\mathsf{sk}_i, \mathsf{c}_i)$ :
+As well as the integer child index $i$, the child key derivation function defined here supports a
+byte sequence $\mathsf{tag}$ and a $\mathsf{full\_width\_leaf}$ flag (representing a boolean value
+where $0$ means false and $1$ means true). Each triple $(i, \mathsf{tag}, \mathsf{full\_width\_leaf})$
```

Comment:
To return to my earlier comment, what's not clear to me from the documentation here is what the `true` and `false` values of `full_width_leaf` signify. What does it mean to be "full-width"?